packages/webdriver/src/command.ts

```diff
@@ -99,6 +101,10 @@ export default function (
             }
 
             this.emit('result', { method, endpoint, body, result })
+
+            if (commandUsage === DELETE_SESSION_COMMAND) {
```

Comment:
Why not just?
```suggestion
            if (command === 'deleteSession') {
```

Reply:
I'm sorry for the delay, I think we should use `deleteSession()` instead of `deleteSession` here, because we get the `deleteSession()` command string.

![Screen Shot 2020-11-09 at 09 29 56](https://user-images.githubusercontent.com/33267199/98507267-4c7dec80-226e-11eb-96d2-ca2e517df682.png)